src/dispatch/dispatch.cpp

```diff
@@ -55,9 +56,10 @@ struct command create_cmd             = {"create", {"--project"}, true, true};
 struct command logs_cmd               = {"logs", {"-n", "--tail"}, true, true};
 struct command fetch_cmd              = {"fetch", {"--table"}, true, true};
 struct command kill_deployment_cmd    = {"kill", {"--deployments", "--job_ids"}, true, true};
-struct command remote_cmd             = {"remote", {}, true, true};
+struct command remote_cmd             = {"remote", {"--default"}, true, true};
```

Comment:
Maybe consider changing the default flag from `--default` to `--set-default` for more clarification, May not be necessary especially if command description is added

Reply:
Made this change in #41 in 30591ea